libraries/libfc/include/fc/variant.hpp

```diff
@@ -362,8 +359,8 @@ namespace fc
         void    clear();
       private:
         void    init();
-        double  _data;                ///< Alligned according to double requirements
-        char    _type[sizeof(void*)]; ///< pad to void* size
+        //enough room to store pointers, doubles, uint64s. doubled to allow 1 extra byte to store type at the end
+        alignas(double) std::array<char, std::max(sizeof(uintmax_t ), sizeof(double)) * 2> _data = {};
```

Comment:
afaict we just need a +1 but I doubled it so it fits nicely in 16 byte register (and 16 bytes is what it was previously too)